ts/components/dialog/debug/components.tsx

```diff
@@ -81,130 +78,77 @@ export const DebugActions = () => {
           }}
         >
           <SessionSpinner loading={loadingLatestRelease} color={'var(--text-primary-color)'} />
-          {!loadingLatestRelease ? 'Check latest release' : null}
+          {!loadingLatestRelease ? 'Check stable version' : null}
         </SessionButton>
-      </Flex>
-    </>
-  );
-};
-
-const unsupportedFlags = ['useTestNet'];
-const untestedFlags = ['useOnionRequests', 'useClosedGroupV3', 'replaceLocalizedStringsWithKeys'];
-
-const handleFeatureFlagToggle = async (
-  forceUpdate: () => void,
-  flag: SessionFeatureFlagsKeys,
-  parentFlag?: SessionFeatureFlagsKeys
-) => {
-  const currentValue = parentFlag
-    ? (window as any).sessionFeatureFlags[parentFlag][flag]
-    : (window as any).sessionFeatureFlags[flag];
-
-  if (parentFlag) {
-    (window as any).sessionFeatureFlags[parentFlag][flag] = !currentValue;
-    window.log.debug(`[debugMenu] toggled ${parentFlag}.${flag} to ${!currentValue}`);
-  } else {
-    (window as any).sessionFeatureFlags[flag] = !currentValue;
-    window.log.debug(`[debugMenu] toggled ${flag} to ${!currentValue}`);
-  }
-
-  forceUpdate();
-};
-
-const FlagToggle = ({
-  forceUpdate,
-  flag,
-  value,
-  parentFlag,
-}: {
-  forceUpdate: () => void;
-  flag: SessionFeatureFlagsKeys;
-  value: any;
-  parentFlag?: SessionFeatureFlagsKeys;
-}) => {
-  const key = `feature-flag-toggle${parentFlag ? `-${parentFlag}` : ''}-${flag}`;
-  return (
-    <Flex
-      key={key}
-      id={key}
-      container={true}
-      width="100%"
-      alignItems="center"
-      justifyContent="space-between"
-    >
-      <span>
-        {flag}
-        {untestedFlags.includes(flag) ? <HintText>Untested</HintText> : null}
-      </span>
-      <SessionToggle
-        active={value}
-        onClick={() => void handleFeatureFlagToggle(forceUpdate, flag, parentFlag)}
-      />
-    </Flex>
-  );
-};
+        <SessionButton
+          onClick={async () => {
+            const userEd25519SecretKey = (await UserUtils.getUserED25519KeyPairBytes())
+              ?.privKeyBytes;
+            if (!userEd25519SecretKey) {
+              window.log.error('[debugMenu] debugAlphaRelease no userEd25519SecretKey');
+              setLoadingAlphaRelease(false);
+              return;
+            }
+            setLoadingAlphaRelease(true);
+            const result = await getLatestReleaseFromFileServer(userEd25519SecretKey, 'alpha');
+            if (!result) {
+              ToastUtils.pushToastError('debugAlphaRelease', 'Failed to fetch alpha release');
+              setLoadingAlphaRelease(false);
+              return;
+            }
+            const [versionNumber, releaseChannel] = result;
+            if (!versionNumber) {
+              ToastUtils.pushToastError(
+                'debugAlphaRelease',
+                `Failed to fetch ${releaseChannel} release`
+              );
+              setLoadingAlphaRelease(false);
+              return;
+            }
+            setLoadingAlphaRelease(false);
```

Comment:
could you extract that button to a component in the same file, and bring its use with it?